bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,111 @@
+# Stellar accounts as a service
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 2000 XLM | 800 XLM | Continuous
+
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+stellar-accounts-as-a-service+) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&title=%F0%9F%94%B5+%60stellar-accounts-as-a-service.md%60) this bounty.
+
+## Description
+
+### What is this task?
+
+Each Stellar account needs to hold 1 XLM to be valid. This can be costly when companies like exchanges want to build
+on Stellar.
```

Comment:
According to the [brand policy](https://www.stellar.org/brand-policy) this needs to be changed:
```suggestion
on the Stellar network.
```
![grafik](https://user-images.githubusercontent.com/1752217/128659892-e8a4ee92-5005-4ed3-8591-cc6fcde21b5c.png)

Reply:
Damn nice catch.